common/primitives/Cargo.toml

```
@@ -16,8 +16,8 @@ targets = ['x86_64-unknown-linux-gnu']
 codec = { package = "parity-scale-codec", version = "3.2.2", default-features = false, features = [
```

Parity has updated their codec.
```suggestion
codec = { package = "parity-scale-codec", version = "3.4.0", default-features = false, features = [
```

---

Hmm there is a mismatch for this crate between substrate and polkadot repos.
 I've checked the substrate repo `polkadot-v0.9.43` branch and  for `parity-scale-codec` the version is "3.2.2"
but on polkadot repo `release-v0.9.43` branch the version is "3.4.0" and on cumulus its "3.1.5"
Not sure which one would make sense for us. Open to any ideas.

---

With v1.0.0 we'll move to `v3.6.1` for this package, so I think it is fine as is. Added a note to the v1.0.0 upgrade issue.